RS: 7.22.0 release docs (#1350)
redis/docs: content/operate/rs/references/rest-api/requests/actions/_index.md

Regardless of an action’s source, each action contains the following attributes: `name`, `action_uid`, `creation_time`, `status`, and `progress`.

#### Example JSON body

```json
{
    "action_type": "task",
    "action_uid": "6155403f-c26f-40ab-8afc-23ed663973f6",
    "additional_info": {
        "object_type": "node",
        "object_uid": "1"
    },
    "creation_time": 1742595918,
    "name": "retry_bdb",
    "progress": 100.0,
    "status": "completed"
}
```

### Status codes {#get-status-codes-v2}

| Code | Description |
|------|-------------|
| [200 OK](https://www.rfc-editor.org/rfc/rfc9110.html#name-200-ok) | No error, response provides info about an ongoing action |

> | [404 Not Found](https://www.rfc-editor.org/rfc/rfc9110.html#name-404-not-found) | Action does not exist (i.e. not currently running and no available status of last run) |

Reviewer: ```suggestion
| [404 Not Found](https://www.rfc-editor.org/rfc/rfc9110.html#name-404-not-found) | Action does not exist (that is, not currently running and no available status of last run) |
```
```suggestion
| [404 Not Found](https://www.rfc-editor.org/rfc/rfc9110.html#name-404-not-found) | Action does not exist (i.e. not currently running and no available status of last run) |
```
